README.md

```diff
@@ -25,3 +30,77 @@ Home Assistant component that interfaces FEMS and OpenEMS, mainly used in Feneco
 2. Manually add this repository to HACS
 4. Add FEMS Integration
 6. Enter your OpenEMS / FEMS address and user account (Fenecon standard:  x / user)
+
+The Fenecon password (user or owner) controls your access rights. If you just want to monitor (read-only access), "user" is fine. If you want to change settings, you need to use "owner".
+
+## Installation
+
+After installation most devices and entities are disabled. You can enable them like this:
+
+<img src="https://github.com/user-attachments/assets/3c4619fa-41a7-4b20-a4f5-46b1fd692cfc" width="750">
+
+
+Some devices to watch for:
+
+1. ctrlIoHeatPump0
+2. meter0
+3. timeOfUseTariff0
+4. batteryInverter0
+5. charger0 and charger1 (these are inverter strings)
+
+## Entites
+
+### _ssum
+The most relevant entities (for most people) are in the _sum device:
+
+<img src="https://github.com/user-attachments/assets/95507715-3e03-43f3-ae13-e46cf0ffc5e2" width="200">
+
+Here you can find your power production, consumption, battery charge and many more.
+
+### Charger0 and 1
+Here you can find voltage, current and power for both of your strings.
+
+<img src="https://github.com/user-attachments/assets/9d09b670-0d72-44dc-b905-f1b5cb0097f3" width="200">
+
+### BatteryInverter
+Has the temperatures for radiator, air, ...
+
+* _batteryinverter0_bmspacktemperature
+* _batteryinverter0_airtemperature
+* _batteryinverter0_radiatortemperature
+
+And the state of health
+* sensor.fems63566_batteryinverter0_bmssoh
+
+<img src="https://github.com/user-attachments/assets/d8e0df40-8a91-41d9-b04f-3b9e1d402193" width="200">
+
+### ctrlIoHeatPump0
+Here you will find the current state of your SG ready connected heatpump.
+
+<img src="https://github.com/user-attachments/assets/c4f8944f-7262-4b35-8d5c-00d4e9eb61ba" width="200">
+
+### Wallbox
```

Comment:
Same thing here, names could differ. Mine is `evcs0` which seems to be set during the installation process? Maybe just indicate that in the documentation so people can find their own entries.

This name might change in the future too, openEMS is working on a new implementation where the naming of the device will change.

Reply:
Just checked the screenshot, looks like the control device which is named `ctrlEvcs0` for me :)